### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#249] [Releases]
## What's Changed
* ⚒️ Fix favicon by @06Games in https://github.com/alexjustesen/speedtest-tracker/pull/237
* 🎉 Search and select multiple servers by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/216
* 👀 Percent change on stats widgets by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/247
* Release 0.9.0 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/248

## New Contributors
* @06Games made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/237

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.8.1...v0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.9.0'>v0.9.0</a>.</em>

**Comment by hungtrungthinh** [comment]
How can I manually install without docker?? Native with Laravel only.

**Reply by alexjustesen** [reply]
You can clone the repo to a VM and follow the directions below for a Laravel deployment. The only specific requirements (I think because I'm on cup of coffee 1) are...

1. Install Speedtest CLI https://www.speedtest.net/apps/cli
2. Setup a cron task to run the scheduler
3. Setup a queue worker against the `default` queue

https://laravel.com/docs/9.x/deployment

**Comment by TobyAnscombe** [comment]
Hey @alexjustesen  - just had the new version installed by Watchtower; Gotta say, nice job on the graphs and overall refinement. 

Well done to you and @06Games for this release, nice Christmas gift ;-)

**Reply by alexjustesen** [reply]
Thanks!
